include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -120,6 +122,49 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
   /// Returns the total number of bytes read since this object was created.
   int64_t ByteCount() const override { return byte_count_ - backup_count_; }
 
+#ifdef GRPC_CORD_SUPPORT_ENABLED
```

Comment:
who's going to define this?

Reply:
Oh I missed this. This is on by default in OSS. This doesn't do anything if it's built with Protobuf prior to v23